File: StykerDG.FarmForge.DataModel/Models/CropVariety.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("CropVariety")]
    public class CropVariety : BaseModel
    {
        [Key]
        public int CropVarietyId { get; set; }
        public int CropTypeId { get; set; }
        public string Name { get; set; }
        public string Label { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/crops/crop_variety.dart
class CropVariety {
  int cropVarietyId;
  int cropTypeId;
  String name;
  String label;

  CropVariety({
    this.cropVarietyId,
    this.cropTypeId,
    this.name,
    this.label
  });

  factory CropVariety.fromMap(Map<String, dynamic> data) {
    return CropVariety(
      cropVarietyId: data['CropVarietyId'],
      cropTypeId: data['CropTypeId'],
      name: data['Name'],
      label: data['Label']
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'CropVarietyId': this.cropVarietyId,
      'CropTypeId': this.cropTypeId,
      'Name': this.name,
      'Label': this.label
    };
  }
}